auth/auth_test.go

```diff
@@ -32,7 +33,7 @@ func TestLoadUserManager(t *testing.T) {
 				Naive:  nil,
 				Github: &g}
 			_, err := LoadUserManager(a)
-			So(err, ShouldBeNil)
+			So(err, ShouldNotBeNil)
```

Comment:
Seems like this should be nil.

Reply:
this is a really silly test of a factory function, and the function looks right and this is the obvious test. I've fixed some of the wording.